### DISCUSSION TITLE
Ways to stay up to date with ESP overall

### DISCUSSION THREAD

**Discussion by JoseMMontoro** [earthspecies/project#42] [Q&A]
Hi,

As somebody that would like to continue contributing to ESP as it grows and evolves, I would really like to have a way to be up to date on the overall direction of the project, major updates, and the project’s areas where either technical or non-technical contributions would have the most impact. 

Reading the issues and PRs on GitHub is part of this, but the overall goals, timelines, resources that apply to ESP don’t seem to be the best fit for GitHub. I wonder if this is something that could be useful for the community to get and stay engaged. Particularly, I wonder if having “open meetings” would be useful.

In addition to just staying up to date, I’m looking for a way to create a sustaining relationship with ESP so that I can contribute long term to the project. With work and the many things that life brings, having some kind of recurrent update as a volunteer would “force” me to put the time that I do want to put in :)

Hopefully this makes sense and please let me know directly if this wouldn’t work :) Just something I wanted to share as a volunteer contributor. Thanks for all your work!

**Comment by JoseMMontoro** [comment]
Btw just watched the Interspecies presentation and it was very useful for this purpose. A maybe broader question might be how ESP wants to engage with volunteers/contributors that are not “core” to the project but could help expand it or bring in new ideas (like my case)

**Comment by radekosmulski** [comment]
@JoseMMontoro these are great thoughts! Thank you very much for sharing them with us 🙏.

We have been having discussions on this internally for quite a while now. We will incorporate your feedback, the perspective that you share is very valuable. This thread will likely become a focal point for our considerations, thank you very much for opening it and for sharing your thoughts. Extremely grateful for your continued contributions to the project 🙂

**Comment by crywolfproject** [comment]
Just an FYI. The link to "on Github" at https://www.earthspecies.org/about is a dead link on Github.
